hypershift-operator/controllers/nodepool_controller.go

```diff
@@ -5,21 +5,20 @@ import (
 	"fmt"
```

Comment:
The event recorder in line 56 should be for "nodepool-controller" https://github.com/openshift-hive/hypershift/pull/31/files#diff-804e9a40d4fe9658dc8326383ec33b1cc577ab536c2b8b6ec58fcbff122a1718L56